content/src/main/content/jcr_root/etc/clientlibs/acs-commons/rte-rtl-switch/.content.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:ClientLibraryFolder"
+    categories="[rte.coralui2]"/>
```

Comment:
Can you change this category to `acs-commons.rte-rtl-switch` (or something similar) so we can follow the pattern outlined in: https://github.com/Adobe-Consulting-Services/acs-aem-commons/issues/864